scripts/image-builder/config/blueprint_v0.0.1.toml

```diff
@@ -43,5 +43,8 @@ version = "*"
 
 [customizations]
 
+[customizations.firewall]
```

Comment:
This change cannot be unconditional. Please, append the firewall settings to the copied blueprint only if prometheus option is selected